tests/handlers/test_oidc.py

```diff
@@ -12,18 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
-from typing import Any, Dict, Tuple
-from unittest.mock import ANY, Mock, patch
+from typing import Any, Awaitable, Dict, Optional, Tuple, TypeVar
+from unittest.mock import ANY, Mock, _patch, patch
```

Comment:
Importing `_patch` feels really dirty (and seems to fail on olddeps), other option is to leave it untyped?